File: src/main/java/com/post_office/BoxHelper.java
package com.post_office;

import java.util.Scanner;

public class BoxHelper {
    public static void main(String[] args) {
        System.out.print("Please enter object's length: ");
        Scanner scanner = new Scanner(System.in);
        float length = scanner.nextFloat();
        System.out.print("Please enter object's width: ");
        float width = scanner.nextFloat();
        System.out.println("Please enter object's height: ");
        int height = scanner.nextInt();

        if(new NO3Box().validate(length , width , height))
            System.out.println("BOX3");
        else if (new NO5Box().validate(length , width , height))
            System.out.println("BOX5");
        else
            System.out.println("None");
    }
}

File: src/main/kotlin/post_office/BoxHelper.kt
package post_office

import com.post_office.NO5Box
import java.util.*

fun main(args: Array<String>) {
    print("Please enter object's length: ")
    var scanner = Scanner(System.`in`)
    var length: Float = scanner.nextFloat()
    print("Please enter object's Width: ")
    var width: Float = scanner.nextFloat()
    print("Please enter object's Height: ")
    var height: Int = scanner.nextInt()

    if (NO3Box().validate(length , width , height))
        print("BOX3")
    else if (post_office.NO5Box().validate(length , width , height))
        print("BOX5")
    else
        print("None")
}